src/System.Net.NameResolution/src/System/Net/NameResolutionPal.Unix.cs

```diff
@@ -13,6 +13,8 @@ namespace System.Net
 {
     internal static partial class NameResolutionPal
     {
+        public static bool SupportsGetAddrInfoAsync => false;
```

Comment:
Can you make this a const?  That would help the compiler to eliminate branches at the call site when doing the unix build.